### PR TITLE
Dear Mary, please review my assignment.

### PR DESCRIPTION
**Pull request by aliakbar0**

### CODE REVIEW

**Comment by aliakbar0** [comment]
Thanks,

File: pacific.css [hunk 1]
```diff
@@ -0,0 +1,209 @@
+header, hgroup, nav, footer, figure, figcaption, aside, section, article {
+    display: block;
+}
+
+body {
+    color: #666666;
+    background-color: #ffffff;
+  f  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
```
Comment (mbMosman) [inline pacific.css]: The extra f here is making you font css invalid.

File: pacific.css [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+    clear: both;
+    margin-left: 190px;
+    background-color: #FFFFFF;
+}
+
+@media only screen and (max-width: 768px) {
+    body { margin: 0; background-image: none; background-color: #ffffff; }
+    #wrapper { min-width: 0; width: auto; box-shadow: none;}
+    nav {float: none; width: auto; padding: .5em; text-align: center;}
+    nav li { display: inline; padding: 0px .75px 0px .75px; }
```
Comment (mbMosman) [inline pacific.css]: The padding should be .75 em not px on the sides and .5em top/bottom, not 0

File: pacific.css [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+
+@media only screen and (max-width: 480px) {
+    h1 { font-size: 1.5em; }
+    main { padding: 0.1em 1em 0.1em; margin: 0; font-size: 90%; }
+    
+    .homehero { background-image: none;height: 0px;}
+	.yurthero { background-image: none;height: 0px;}
+	.trailhero { background-image: none;height: 0px;}
+    
+    nav ul { text-align: center; padding: 0; margin: 0; }
```
Comment (mbMosman) [inline pacific.css]: nav padding should be 0 for mobile.

File: pacific.css [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+    list-style-position: inside;
+    
+}
+
+#contact {
+    font-size: 90%;
+}
+
+.mobile {
+    display: none;
```
Comment (mbMosman) [inline pacific.css]: This should only have the display property, not the background image or height.

**Comment by mbMosman** [comment]
Overall good job.  A few things to fix, but mostly minor.

**Comment by aliakbar0** [comment]
Thanks

**Pull request closed without merging**